praw/exceptions.py

```diff
@@ -37,10 +37,26 @@ def __init__(
         :param error_type: The error type set on Reddit's end.
         :param message: The associated message for the error.
         :param field: The input field associated with the error, if available.
+
+        There are three ways to instantiate this class:
+
+        * .. code-block:: python
```

Comment:
I just want to confirm that this renders code blocks as bullets.

Reply:
Yes it does.